src/utils/entity-utils.ts

```diff
@@ -33,3 +34,32 @@ export const generateDisabledSpec = (
         }
     }
 };
+
+export const getSourceCapture = (
+    sourceCapture: SourceCaptureDef | string | null | undefined
+): SourceCaptureDef | null => {
+    if (!sourceCapture) {
+        return null;
+    }
+
+    if (typeof sourceCapture === 'string') {
+        return {
+            capture: sourceCapture,
+        };
+    }
+
+    return sourceCapture;
+};
+
+export const addOrRemoveSourceCapture = (
+    draftSpec: any,
+    sourceCapture: SourceCaptureDef | null
+) => {
+    if (sourceCapture) {
+        draftSpec.sourceCapture = sourceCapture;
+    } else {
+        delete draftSpec.sourceCapture;
```

Comment:
Out of curiosity, do you prefer `delete` to Lodash's `omit`?

Reply:
Yeah